docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -204,6 +396,21 @@ quarkus:
       ~: true
       methods: GET,PUT,POST
 ----
-
-YAML `null` keys are not included in the assembly of the configuration property name, allowing them to be used
-in any level for disambiguating configuration keys.
++
+. To compile your {project-name} application in development mode, enter the following command from the project directory:
++
+.Compile your {project-name} application
+[source,bash]
+----
+./mvnw quarkus:dev
+----
++
```

Comment:
I'm not sure of the value of this here?

Reply:
Generally, this type of information is useful for non-expert users such as students. Red Hat generally gets complaints from newer users that our documentation assumes too much expertise or prior knowledge.